src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -51,6 +51,15 @@
   <input type="checkbox" class="sidebar-toggle" name="__secondary" id="__secondary">
   <label class="overlay overlay-secondary" for="__secondary"></label>
 
+  <!-- A search field pop-up that will only show when the search button is clicked -->
+
+  <div class="search-button__wrapper">
+    <div class="search-button__overlay"></div>
+    <div class="search-button__search-container">
```

Comment:
previously this was `{{ containerClass }}`, I'm having trouble figuring out the reasons / implications of hard-coding it instead.  Is it easy for you to explain?

Reply:
the main reason I removed it is because it only seemed to be used in a single place in all of the theme, so I suspect in a previous iteration we had used it in multiple places, but then had removed one of those places. If we're only using it once, and if we [were hard-coding it as a variable](https://github.com/pydata/pydata-sphinx-theme/pull/832/files#diff-84e18f5af7335d9589b586c8cc7aa3e960c62d8429620dfef32e341d8be1da70L3), I figured it'd be simpler to just remove the variable.

Reply:
makes sense, thanks!